web_source/cities/south_america/argentina/cordoba.md

```diff
@@ -9,7 +9,7 @@ koɾðoβa
 ### What we would like to see there
 
 If you're into religion, its got a really lovely church
-![image of a church](https://commons.wikimedia.org/wiki/File:Catedral_de_C%C3%B3rdoba,_Argentina_3.jpg)
+![image](https://commons.wikimedia.org/wiki/File:Catedral_de_C%C3%B3rdoba,_Argentina_3.jpg)
```

Comment:
This link is to the HTML page with info about the image, not the image itself, which is why it doesn't work. Descriptive alt-text is also good ;)
```suggestion
![Photo of a cathedral](https://upload.wikimedia.org/wikipedia/commons/thumb/3/36/Catedral_de_C%C3%B3rdoba%2C_Argentina_3.jpg/64px-Catedral_de_C%C3%B3rdoba%2C_Argentina_3.jpg?20060610023555)```